sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/cloud/core/resource/AzureStorageBlobProtocolResolver.java

```diff
@@ -39,12 +39,6 @@ public final class AzureStorageBlobProtocolResolver extends AbstractAzureStorage
         RETRIEVE_NOTHING_CONTAINER_DETAILS.setRetrieveDeleted(false);
     }
 
-    /**
-     * The default constructor of AzureStorageBlobProtocolResolver.
-     */
-    public AzureStorageBlobProtocolResolver() {
-    }
-
```

Comment:
This is why, we delete the ctor.

Reply:
We can use the default ctor. And we do not have other ctors in this class.  To keep it clean as suggested from Sonar.